sdcm/remote.py

```diff
@@ -67,7 +67,7 @@ def run(self, command, **kwargs):
     @opens
     def create_session(self):
         channel = super(ConnectionCmdTimeout, self).create_session()
-        if self.cmd_timeout:
+        if hasattr(self, 'cmd_timeout') and self.cmd_timeout:
```

Comment:
do we still need the `hasattr` check after you are assigning the class atribute "Invoke" way?

Reply:
i think it could be removed